Models/studentModels/select.ts

```diff
@@ -0,0 +1,9 @@
+import { executeQuery } from '../../utils/database';
+import { StudentProps } from '../../Interfaces/studentinterface';
+
+export const studentSelect = async (): Promise<StudentProps[]> => {
+    const selectAllStudents = "SELECT * FROM `Student`";
```

Comment:
table name should be started with uppercase

Reply:
Already done